### PR TITLE
Enable more options on CLI

### PR DESCRIPTION
**Pull request by JWZ2018**
Issue: #804
    Summary: Enable timeout and additionalFunctions on the CLI
    Test Plan:
    Added options on the CLI and used a debugger to confirm values are passed into Prepack

### CODE REVIEW

**Comment by JWZ2018** [comment]
These changes only add in the options that are missing in the CLI that should be there.

If the enforcing equivalence part is still needed, I can restructure the argument parsing logic in the CLI so that the equivalence can be checked even when not running from the CLI.

File: src/prepack-cli.js [hunk 1]
```diff
@@ -43,7 +43,9 @@ function run(
     --srcmapIn          The input sourcemap filename. If present, Prepack will output a sourcemap that maps from
                         the original file (pre-input sourcemap) to Prepack's output
     --srcmapOut         The output sourcemap filename.
-    --maxStackDepth     Specify the maximum call stack depth
+    --maxStackDepth     Specify the maximum call stack depth.
+    --timeout           The amount of time until Prepack should time out.
+    --addtlFunctions    Additional functions that should be prepacked (comma separated).
```
Comment (NTillmann) [inline src/prepack-cli.js]: No funky abbreviations please, especially if they don't even match the actual parameter name!

Comment (JWZ2018) [reply src/prepack-cli.js]: Oops! Fixed

File: src/prepack-cli.js [hunk 2]
```diff
@@ -36,26 +36,28 @@ function run(
...
-                        and instead inline them where they are used.
+    input                    The name of the file to run Prepack over (for web please provide the single js bundle file)
+    --out                    The name of the output file
+    --compatibility          The target environment for Prepack [${CompatibilityValues.map(v => `"${v}"`).join(", ")}]
+    --mathRandomSeed         If you want Prepack to evaluate Math.random() calls, please provide a seed.
+    --srcmapIn               The input sourcemap filename. If present, Prepack will output a sourcemap that maps from
+                             the original file (pre-input sourcemap) to Prepack's output
+    --srcmapOut              The output sourcemap filename.
+    --maxStackDepth          Specify the maximum call stack depth.
+    --timeout                The amount of time until Prepack should time out.
```
Comment (NTillmann) [inline src/prepack-cli.js]: In seconds or what?

**Pull request closed without merging**